app/controllers/questions_controller.rb

```diff
@@ -81,6 +81,23 @@ def destroy
     end
   end
 
+  # This method is used when a student answers a question and submits the form.
+  def submit_question
+    updated = false
+    if params[:question] && params[:answer]
```

Comment:
This resolves the app breaking when a user submits the empty form. Kinda sloppy right now. Will refactor.